test/transaction/test_transaction.js

```diff
@@ -0,0 +1,121 @@
+import test from 'ava'
+import { Transaction, Ed25519Keypair } from '../../src'
+import { _makeTransferTransaction } from '../../src/transaction/makeTransferTransaction'
+import makeInputTemplate from '../../src/transaction/makeInputTemplate'
+
+
+// TODO: Find out if ava has something like conftest, if so put this there.
+const alice = new Ed25519Keypair()
+const aliceCondition = Transaction.makeEd25519Condition(alice.publicKey)
+const aliceOutput = Transaction.makeOutput(aliceCondition)
+const assetMessage = { assetMessage: 'assetMessage' }
+const metaDataMessage = { metaDataMessage: 'metaDataMessage' }
+const createTx = Transaction.makeCreateTransaction(
+    assetMessage,
+    metaDataMessage,
+    [aliceOutput],
+    alice.publicKey
+)
+const transferTx = Transaction.makeTransferTransaction(
+    createTx,
+    metaDataMessage,
+    [aliceOutput],
+    0
+)
+
+
+test('Create valid output with default amount', t => {
+    const condition = {
+        details: {
+            public_key: 'abc'
+        }
+    }
+    const expected = {
+        amount: '1',
+        condition,
+        public_keys: ['abc']
+    }
+    const res = Transaction.makeOutput(condition)
+    return t.deepEqual(res, expected)
```

Comment:
I don't think it is. Thx. I'll fix, also [here](https://github.com/bigchaindb/js-bigchaindb-driver/pull/35/files/3389726cb27d44acc543941ef8344432985ed7b5#diff-69db316883ecc07bb2ce9c436a44a45c)